cdb2api/cdb2api.h

```diff
@@ -89,6 +89,7 @@ enum cdb2_errors {
     CDB2ERR_MALLOC = 115,
     CDB2ERR_NOTSUPPORTED = 116,
 
+    CDB2ERR_SCHEMACHANGE = 240,
```

Comment:
Can we reuse CDB2ERR_SCHEMA instead of adding a new code?

Reply:
`CDB2ERR_SCHEMA` indicates that the table version has changed.